responsemanager/server.go

```diff
@@ -124,6 +155,9 @@ func (rm *ResponseManager) abortRequest(p peer.ID, requestID graphsync.RequestID
 		})
 		delete(rm.inProgressResponses, key)
 		response.cancelFn()
+		if response.span.IsRecording() {
+			response.span.End()
```

Comment:
again, I think we may have a problem with the defer() above in terms of parent/child span closing

Reply:
FOLLOW-UP: This is not a place to end the span see https://github.com/ipfs/go-graphsync/issues/288